package.json

```diff
@@ -34,6 +34,7 @@
 		"new:package": "plop --plopfile scripts/new-package.mjs && yarn format && yarn manypkg fix && preconstruct fix && yarn install",
 		"playroom:build": "yarn --cwd docs playroom:build",
 		"playroom:dev": "yarn --cwd docs playroom:dev",
+		"prepublishOnly": "echo 'Publishing from local machine is disabled. Use CI/CD pipeline.' && echo 'Push changes to develop branch to trigger release process.' && exit 1",
```

Comment:
is this comment incorrect? releases are triggered from main and changes are merged to develop. perhaps we should update RELEASE.md and refer to it here.